### ISSUE TITLE
Users unclear on steps to move forward on single cell workbench:

### ISSUE THREAD

**Issue by adkinsrs** [IGS/gEAR#175]
Users unclear on steps to move forward on single cell workbench:

In the single-cell workbench, after users select a dataset and run the first filtering step, they were unclear on that they had to click the next toggle to move on. Perhaps naming the steps in order (1,2,3,etc) or including some text that pops up after they apply filters that they need to click the next toggle would be useful.

<img width="498" alt="Screen Shot 2022-01-19 at 10 28 00 AM" src="https://user-images.githubusercontent.com/28451557/150162574-b2eafd9a-f317-479a-93ba-4ba985cf0c6e.png">

_Originally posted by @JPReceveur in https://github.com/IGS/gEAR/discussions/168#discussioncomment-1998801_

**Comment by adkinsrs** [comment]
I am going to add a "Please click the <step> toggle" message at the bottom of the current step's panel that will trigger to show when the current step is complete.  However I will not add numbers to the steps on the list of toggles on the left, for the reason that loading from existing saved analyses only uses some of the steps and thus would require a renumbering of the steps.

**Comment by adkinsrs** [comment]
I'm testing this but ran into a bit of a roadblock.  Back in December, I had to update my Docker image due to various python packages that I was using being changed in their backversions, which broke some things. I got everything working except for the "louvain" package which is used by scanpy, so I skipped it in order to fix it later.  Well "later" is now, since the louvain steps are in the analysis workbench, and I cannot run that step on my Docker instance.
